moveit_ros/moveit_servo/include/moveit_servo/servo_node.h

```diff
@@ -63,6 +63,12 @@ class ServoNode
   std::shared_ptr<tf2_ros::Buffer> tf_buffer_;
   std::shared_ptr<planning_scene_monitor::PlanningSceneMonitor> planning_scene_monitor_;
 
+  /***
+   * \brief Most of parameters are validated using the validation methods provided by the generate_paramter_library
+   * This method validates only the parameters that need to be checked against another parameter
```

Comment:
Recommend taking some extra time in documenting things and checking for typos and punctuation -- there is no hurry to turn around feedback, and it takes longer to go back and forth multiple times in the long run.

This is just an idea:

```suggestion
   * \brief Most servo parameters are individually validated using the validation methods in `generate_parameter_library`, with limits specified in the parameters YAML file.
   * This method performs additional validation for parameters whose values depend on each other. 
```

Reply:
Got it, will take care.
Thanks.